systemd/user/kano-common.target

```diff
@@ -13,4 +13,5 @@ Description=Kano Common User Services
 IgnoreOnIsolate=true
 Wants=kano-common-tracker.service kano-common-notifications.service \
  kano-boards.service kano-speakerleds.service \
- kano-common-keyboard.service kano-home-button.service
+ kano-common-keyboard.service kano-home-button.service \
+ kano-common-track-space.service
```

Comment:
I would [Install] this service via the default target, e.g. https://github.com/KanoComputing/kano-settings/blob/master/systemd/system/check-enable-ssh.service#L20 

This may then require an enable/disable in postinst/postrm, e.g. https://github.com/KanoComputing/kano-desktop/blob/master/debian/kano-desktop.postinst#L128

Reply:
Are you saying that it should start before the user logs in? The tracker isn't running then.